Realm/Realm.Weaver/Analytics/AnalyticsUtils.cs

```diff
@@ -272,45 +269,6 @@ public static string GetAnonymizedUserId()
             }
         }
 
-        // This is just a placeholder, until we figure out how to extract the proper info from Unity
-        public static string ConvertUnityArchitectureToMetricsVersion(int unityArchitecture)
-        {
-            var convertedValue = unityArchitecture switch
-            {
-                0 => "None",
-                1 => CpuArchitecture.Arm64,
-                2 => "Unity_Universal",
-                _ => Unknown(System.Environment.OSVersion.Platform.ToString())
-            };
-
-            return Unknown(convertedValue);
-        }
-
-        private static string ConvertArchitectureToMetricsVersion(string arch)
```

Comment:
If I recall correctly the reason for this to exist was that we trusted what Mono.Cecil was telling us about the assembly architecture and it uses i386 or amd64. Since we don't use that info anymore I guess it's good for this to go.